tests/framework/Optimizers/Stochastic/stoch_parabola.py

```diff
@@ -17,7 +17,8 @@ def random(scale=0.5,loc=-1.0):
   return scale*(2.*np.random.rand()+loc)
 
 def evaluate(values):
-  return base(values) + random()
+  ran = random()/10.0
```

Comment:
In devel the test was not converging, but just reaching the maximum number of iterations (the screenshot above is taken from running the test from ```devel```)